cli/functionality/team23.go

```diff
@@ -0,0 +1,34 @@
+/*
+This file contains extra functionality for CLI commands. Make sure to 
+capitalize first letter of functions to be used in external folders.
+*/
+
+package functionality
+
+import (
+	"fmt"
+	"os"
+)
+
+
+func test(input string) (int){
```

Comment:
It is common to have the return value, `int`, in parenthesis? Might be good for all of us to come to an agreement on how to format it